File: Python/functions/no checking/twin prime.py
def is_prime(n):
    for i in range(2,n):
        if n%i==0:
            return False
    return True
n1 = int(input("Enter the 1st No. = "))#n1=4
n2 = int(input("Enter the 2nd No. = "))#n2=7
if is_prime(n1) and is_prime(n2) and abs(n1 - n2) == 2:
    print("Twin Prime")
else:
    print("Not Twin Prime")

File: C Programms/No. Checking/twin_prime.c
//WAP to check Twin prime no.
#include<stdio.h>
int is_prime(int n)
{
    for(int i=2;i<n;i++)
    {
        if(n%i==0)
            return 0;
    }
    return 1;
}
void main()
{
    int a,b;
    printf("Enter two no. : ");
    scanf("%d%d",&a,&b);
    ((is_prime(a)==1) && (is_prime(b)==1) && (a-b == 2 || b-a == 2))?printf("%d and %d are Twin Prime no.",a,b):printf("%d and %d are not Twin Prime no.",a,b);

}